image.go

```diff
@@ -557,6 +557,10 @@ func (i *containerImageSource) LayerInfosForCopy(ctx context.Context) ([]types.B
 	return nil, nil
 }
 
+func (i *containerImageSource) HasThreadSafeGetBlob() bool {
+	return false
```

Comment:
is this and blobcache ok to put to false? (haven't read the full code)

Reply:
For pulling yes. It should not make any difference for other cases.